android/cli/commands/_build.js

```diff
@@ -4104,6 +4066,10 @@ AndroidBuilder.prototype.compileJavaClasses = function compileJavaClasses(next)
 
 	classpath[path.join(this.platformPath, 'lib', 'titanium-verify.jar')] = 1;
 
+	if (this.encryptJS) {
+		classpath[path.join(this.platformPath, 'ti.cloak.jar')] = 1;
```

Comment:
Just curious why this JAR isn't under the `lib` folder like titanium-verify is...